sklearn/mixture/gaussian_mixture.py

```diff
@@ -499,14 +504,23 @@ class GaussianMixture(BaseMixture):
         The mean of each mixture component.
         `means_` will not exist before a call to fit.
 
-    covariances_ : array
-        The covariance of each mixture component.
-        The shape depends on `covariance_type`::
+    precisions_cholesky_ : array
+        The cholesky decomposition of each mixture component.
+        The shape depends on `precision_type`::
             (n_components,)                        if 'spherical',
             (n_features, n_features)               if 'tied',
             (n_components, n_features)             if 'diag',
             (n_components, n_features, n_features) if 'full'
-        `covariances_` will not exist before a call to fit.
+        `precisions_cholesky_` will not exist before a call to fit.
+
+    precisions_ : array
+        The precision of each mixture component.
+        The shape depends on `precision_type`::
+            (n_components,)                        if 'spherical',
+            (n_features, n_features)               if 'tied',
+            (n_components, n_features)             if 'diag',
+            (n_components, n_features, n_features) if 'full'
+        `precisions_` will not exist before a call to fit.
```

Comment:
This is the case for any attribute that ends in `_` in scikit-learn so I would just remove the sentence "precisions_ will not exist before a call to fit." from this docstring.